### PR TITLE
Resell token functionality

### PR DESCRIPTION
**Pull request by Space-zee**
https://app.clickup.com/t/860ptznyc

### CODE REVIEW

File: contracts/contracts/v3/KycStore.sol [hunk 1]
```diff
@@ -39,13 +40,17 @@ contract KycStore is Initializable {
...
-        return 6;
+        return 7;
     }
 
     function isOperable(address userAddress) public view returns (bool) {
+        ResellPoolFactory resellPoolFactory = ResellPoolFactory(addressesProvider.getResellPoolFactory());
         if (!isKycEnabledInTheApp) {
             return true;
         }
+        if (resellPoolFactory.poolByAsset(userAddress) != address(0)) {
```
Comment (ruslanpalagin) [inline contracts/contracts/v3/KycStore.sol:51]: this is NOT related to KYC so it should not be here
Please move into Asset or where is more related

**Review by ruslanpalagin — CHANGES_REQUESTED** [review]

**Comment by ruslanpalagin** [comment]
please PM me once ready

File: contracts/contracts/v3/KycStore.sol (hunk 1 above)
Comment (Space-zee) [reply contracts/contracts/v3/KycStore.sol]: +

**Review by ruslanpalagin — APPROVED** [review]
